custom/icds_reports/tasks.py

```diff
@@ -747,7 +747,8 @@ def _find_stagnant_cases(adapter, latest_datetime):
     query_object = adapter.get_query_object()
     return (
         query_object.with_entities(table.c.doc_id, table.c.inserted_at).filter(
-            table.c.inserted_at >= latest_datetime
+            table.c.inserted_at >= latest_datetime,
+            table.c.inserted_at <= datetime.utcnow()  # This filter is used to force postgres to use the index
```

Comment:
is this something that is needed each time we filter on a datetime format index?

Reply:
honestly I'm not sure. in investigating the agg querires,  we've recently noticed some issues with our postgres statistics being "off" or at least not producing the queries that we expect. Its possible that this is just hitting some strange edge case with postgres query planner